lib/session.js

```diff
@@ -108,6 +109,9 @@ var doSignin = function(lookup, req, res, onSuccess, onFail) {
 			if (user && (!passwordCheck || scmp(passwordCheck, hash(user.password)))) {
 				postHookedSigninWithUser(user, req, res, onSuccess, onFail);
 			} else {
+				if (!err) {
+					err = new Error('The password you entered is incorrect.');
```

Comment:
Created https://github.com/keystonejs/keystone/issues/1448 to discuss possible security issue